AzurePullRequests.Restore/Services/AzureDevOpsRestoreService.cs

```diff
@@ -52,15 +51,12 @@ public async Task RestorePullRequestsAsync()
                 var jsonPR = File.ReadAllText(pullRequestFileLocation);
                 var pullRequest = JsonConvert.DeserializeObject<GitPullRequest>(jsonPR);
 
-                // check if PR is in DevOps. If there update PR else create new
                 if (await IsPullRequestInDevOps(pullRequestState.Id))
                 {
-                    // update existing
                     await RestoreDevOpsEntityAsync(RestoreType.Update, apiUrl, pullRequest);
                 }
                 else
                 {
-                    // create new
```

Comment:
Starting a review
![images](https://github.com/user-attachments/assets/175a03ac-8dc7-40db-bee7-3b952ef92ee0)

Reply:
A `thread` in a review @ralph0211 
![images](https://github.com/user-attachments/assets/efa11396-d9b0-4fbb-bf7b-2256700c1994)

Reply:
_Another_ review 

> comment before finishing

 the review